plugin/src/Main.server.lua

```diff
@@ -55,15 +55,17 @@ local function main()
 				end)
 		end)
 
-	toolbar:CreateButton("Sync In", "Sync into Roblox Studio", Config.icons.syncIn)
-		.Click:Connect(function()
-			checkUpgrade()
+	local function syncIn()
+		checkUpgrade()
 
-			pluginInstance:syncIn()
-				:catch(function(err)
-					warn(err)
-				end)
-		end)
+		pluginInstance:syncIn()
+			:catch(function(err)
+				warn(err)
+			end)
+	end
+
+	toolbar:CreateButton("Sync In", "Sync into Roblox Studio", Config.icons.syncIn).Click:Connect(syncIn)
+	plugin:CreatePluginAction("RojoSyncIn", "Sync In Rojo Project", "Pulls the connected Rojo project into Studio.").Triggered:Connect(syncIn)
```

Comment:
Looks like this line is too long based on the Luacheck policy we've got, and I'm inclined to agree.

Can we take the strings out and put them into constants? I think the 'Sync In Rojo Project' part should probably be the same between the button and the action.